### PR TITLE
"Show notes" give invalid tab error if notes tab is closed #1643

### LINKED ISSUE #1643: "Show notes" give invalid tab error if notes tab is closed
### Prerequisites:
Mic permissions are enabled.
"Allow" or "Don't Allow" button from “Allow Firefox Voice to Collect Voice Transcriptions” pop-up is clicked.
An online notepad tab is made the default tab where notes are added with the "Make notes here" command. Another tab is then made active and the notes tab is closed.

### STR:
1. Use the shortcut or click on the mic icon from the browser toolbar.
2. Say or write the "Show notes" command.

### Expected result:
The default tab where notes are added opens without issues.

### Actual result:
"Internal error: Error: Invalid tab ID: 6" is displayed in the doorhanger.

### Notes:
Reproduced on Mac 10.14 and Windows 10x64 with Firefox Nightly 78.0a1 (2020-05-12).
The number from the error message may be different depending on the tab used.
If the default tab where notes are added is closed and an "Add note" command is made, "The writing tab has changed, use "show notes" and "write notes here"" is displayed in the doorhanger.

### PR DESCRIPTION
**Pull request by AbhiVaidya95**
Before submitting a final PR, please:

- [✗] Run `npm test` on your machine
- [✓] Run `npm run format` to keep code formatted consistently
- [✓] Reference the issue you are fixing in at least one of your commit messages, as `Fixes #X`

### CODE REVIEW

**Comment by AbhiVaidya95** [comment]
@ianb I'm not sure why my previous commit are listed.Thanks

**Comment by AbhiVaidya95** [comment]
@ianb I made changes to go as you asked, Can you review my changes.

File: extension/intents/notes/notes.js [hunk 1]
```diff
@@ -82,7 +82,14 @@ intentRunner.registerIntent({
       e.displayMessage = "You have not set a tab to write";
       throw e;
     }
-    await browserUtil.makeTabActive(writingTabId);
+    const available = await content.hasScript(writingTabId, SCRIPT);
```
Comment (ianb) [inline extension/intents/notes/notes.js:85]: I _just_ removed this function yesterday. But really it's enough to just catch the error from `browserUtil.makeTabActive()` – if it raises an exception about an invalid tab ID then the tab is gone. So it's a matter of catching that exception and adding `.displayMessage` to it.

Comment (AbhiVaidya95) [reply extension/intents/notes/notes.js]: @ianb I made the changes as you asked, Thanks for the suggestion :)

**Comment by ianb** [comment]
Looks good, thanks!

**Pull request merged**